app/helpers.php

```diff
@@ -1,6 +1,34 @@
 <?php
 
 /**
+ * Build a CSV from the supplied data.
+ *
+ * @param  array  $data
+ * @return \League\Csv $csv
+ */
+function buildCSV($data)
+{
+    $csv = \League\Csv\Writer::createFromFileObject(new \SplTempFileObject());
+
+    foreach ($data as $row) {
+        $csv->insertOne($row);
+    }
+
+    return $csv;
+}
+
+function correspondence($message = null, $field = null)
+{
+    $correspondence = app('Gladiator\Http\Utilities\Correspondence');
```

Comment:
Oh, other silly style nitpick: it'd be nice to use `::class` here for static analysis goodness.